parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -57,17 +55,23 @@
  */
 public class ParselyTracker {
```

Comment:
Suggestion (💡): Consider removing the unnecessary `this.` from everywhere (even within comments). Only a handful of `this.` is needed and even those can be removed by updating the parameter name, or, we can keep `this.` on them anyway (`siteId`, `parentTimer`, `intervalMillis` and `baseEvent`), but still remove it everywhere else. IMHO, this will increase the readability of this class by a lot.

PS: This is an extra, feel free to ignore.